client/src/cbltest/api/syncgateway.py

```diff
@@ -1228,9 +1230,24 @@ async def get_document(
                 "cbl.document.id": doc_id,
             },
         ):
-            response = await self._send_request(
-                "get", f"/{db_name}.{scope}.{collection}/{doc_id}"
-            )
+            if use_public_api:
```

Comment:
a small nit pick, instead of repeating this code snippet multiple times, it makes sense to add this into `_send_request` function